alerts/openshift-virtualization-operator/CDIDefaultStorageClassDegraded.md

```diff
@@ -2,57 +2,65 @@
 
 ## Meaning
 
-This alert fires when there is no default storage class that supports smart cloning
-(CSI or snapshot-based) or the ReadWriteMany access mode.
+This alert fires when there is no default storage class that supports smart
+cloning (CSI or snapshot-based) or the ReadWriteMany access mode.
+
+A default virtualization storage class has precedence over a default OpenShift
+Container Platform
+storage class for creating a VirtualMachine disk image.
 
 ## Impact
 
 If the default storage class does not support smart cloning, the default cloning
 method is host-assisted cloning, which is much less efficient.
 
-If the default storage class does not support ReadWriteMany, virtual machines (VMs)
-cannot be live migrated.
+If the default storage class does not support ReadWriteMany, virtual machines
+(VMs) cannot be live migrated.
 
-Note: A default OpenShift Virtualization storage class has precedence over a
-default OpenShift Container Platform storage class when creating a
+Note: A default OpenShift Virtualization storage class has precedence
+over a default OpenShift Container Platform storage class when creating a
 VM disk.
 
 ## Diagnosis
 
-1. Get the default OpenShift Virtualization storage class by running the following
-command:
+1. Get the default OpenShift Virtualization storage class by running the
+following command:
 
    ```bash
-   $ oc get sc -o jsonpath='{.items[?(@.metadata.annotations.storageclass\.kubevirt\.io/is-default-virt-class=="true")].metadata.name}'
+   $ export CDI_DEFAULT_VIRT_SC="$(oc get sc -o json | jq -r '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name')"
+   $ echo default_virt_sc=$CDI_DEFAULT_VIRT_SC
    ```
```

Comment:
This step now has two commands; in the docs, we only put one command in each code block so that they can be copied more accurately by clicking a "copy" button. This might not be relevant to the runbooks, though I think we want to align with the downstream docs guidelines as much as possible.

If you separate it into two code blocks, you could do multiple sub-steps. For example...


1. Get the default OpenShift Virtualization storage class by running the following
commands:

   1a. Export the default storage class into an environment variable. For example:
   ```bash
   $ export CDI_DEFAULT_VIRT_SC="$(oc get sc -o json | jq -r '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name')"
   ```
   1b. View the default storage class information by printing the variable definition:
   ```bash
   $ echo default_virt_sc=$CDI_DEFAULT_VIRT_SC
   ```

Or perhaps you could combine it into one command with `&&`

Reply:
the echo is just for visibility of the value saved in $CDI_DEFAULT_VIRT_SC, not need for the diagnosis, maybe removing, let me know if we want to keep it